Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Asynchronous Disaster Recovery Failover and Failback
+weight: 3245
+indent: true
+---
+
+# RBD Asynchronous Disaster Recovery Failover and Failback
+
+## Table of Contents <!-- omit in toc -->
+
+* [Planned Migration](#planned-migration)
+  * [Failover](#failover)
+  * [Failback](#failback)
+* [Disaster Recovery](#disaster-recovery)
+  * [Failover](#failover-abrupt-shutdown)
+  * [Failback](#failback-post-disaster-recovery)
+* [Appendix](#appendix)
+  * [Creating a VolumeReplicationClass CR](#create-a-volume-replication-class-cr)
+  * [Creating a VolumeReplications CR](#create-a-volumereplication-cr)
+  * [Check VolumeReplication CR status](async-disaster-recovery.md#checking-replication-status)
+  * [Backup and Restore](#backup-&-restore)
+
+## Planned Migration
+
+> Use cases: Datacenter maintenance, technology refresh, disaster avoidance, etc.
+
+### Failover
+
+The failover operation is the process of switching production to a
+ backup facility (normally your recovery site). In the case of Failover,
+ access to the image on the primary site should be stopped.
+The image should now be made *primary* on the secondary cluster so that
+ the access can be resumed there.
+
+> :memo: As mentioned in the pre-requisites, periodic or one time backup of
+> the application should be available for restore on the secondary site (cluster-2).
+
+Follow the below steps for planned migration of workload from primary
+ cluster to secondary cluster:
+
+* Scale down all the application pods which are using the
+ mirrored PVC on the Primary Cluster.
```

Comment:
Add reference to how PVC or image is enabled for mirroring.

Reply:
Thanks for the suggestion! Added a [new section](https://github.com/rook/rook/pull/8411/files#diff-e1e52c786ed334c193c928332d3ab3af038df4029d9dae9647ae93d6725160b6R260) for the same in the rbd mirroring guide and also gave the user a pointer in failover and failback doc [here](https://github.com/rook/rook/pull/8411/files#diff-1b0b287cce986a56c15d68e1b16ad5941350b0fee93c11ff1de7c6d829456b05R20). Let me know if I am missing anything else.